src/bin/miri.rs

```diff
@@ -313,6 +313,9 @@ fn main() {
                 "-Zmiri-symbolic-alignment-check" => {
                     miri_config.check_alignment = miri::AlignmentCheck::Symbolic;
                 }
+                "-Zmiri-check-number-initialization" => {
```

Comment:
Could you add docs for this flag to the readme?

Also, the flag should affect raw pointers as well as integers and floats. Not sure how to best reflect that in its name though...

Reply:
Ah, actually looks like we already require raw ptrs to be init. I will have to check if we have a test for that.